api/restHandler/TelemetryRestHandler.go

```diff
@@ -92,3 +93,8 @@ func (handler TelemetryRestHandlerImpl) SendTelemetryData(w http.ResponseWriter,
 	common.WriteJsonResp(w, nil, "success", http.StatusOK)
 
 }
+
+func (handler TelemetryRestHandlerImpl) SigtermEventHandler(w http.ResponseWriter, r *http.Request) {
+	handler.telemetryEventClient.SendSigtermSummaryEvent()
```

Comment:
Read eventType from request. (You should send as "SIG_TERM" from hook. )
If eventType is nil or empty then default eventType should be Summary.